src/assets/images/pokeball/Diveball.svg

```diff
@@ -0,0 +1 @@
3c-23.987 0-43.501 19.514-43.501 43.5 0 23.987 19.514 43.501 43.5 43.501 23.987 0 43.502-19.514 43.502-43.5 0-23.987-19.515-43.501-43.501-43.501zm0 92.644C22.046 98.287 0 76.241 0 49.144 0 22.045 22.046 0 49.144 0s49.143 22.045 49.143 49.144c0 27.097-22.045 49.143-49.143 49.143"/><path style="fill:none;stroke:#000;stroke-width:.267111px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" d="M10.751 69.578c1.12-.046 2.231.22 3.259.649 1.31.545 2.507 1.322 3.653 2.15 1.033.748 2.017 1.559 3.026 2.337 1.067.823 2.246 1.51 3.52 1.96 1.257.443 2.587.636 3.912.738.69.054 1.379.11 2.07.12.807.01 1.608-.158 2.368-.42 1.39-.478 2.647-1.3 3.704-2.32.687-.663 1.353-1.349 2.059-1.993a14.351 14.351 0 0 1 1.942-1.49c1.059-.677 2.175-1.258 3.299-1.82.68-.339 1.364-.67 2.05-.997"/><path style="fill:#fff;stroke-width:0;stroke-linecap:round;stroke-linejoin:round" d="M176.308 237.966c-20.25-3.81-36.711-19.102-42.035-39.049-1.878-7.033-1.997-18.602-.262-25.38 4.996-19.521 20.037-34.564 39.526-39.531 6.783-1.73 18.358-1.608 25.38.267 25.71 6.862 43.19 32.534 39.505 58.02-1.765 12.206-6.753 22.233-15.324 30.805-12.627 12.626-29.763 18.071-46.79 14.868zm19.298-15.3c6.374-1.644 12.048-4.963 17.072-9.988 7.576-7.575 11.092-16.183 11.076-27.117-.014-10.444-3.672-19.505-10.73-26.579-7.25-7.268-14.616-10.575-24.939-11.196-29.846-1.795-49.705 28.86-36.184 55.854 4.322 8.629 13.981 16.377 23.762 19.061 4.765 1.308 14.801 1.29 19.943-.036z" transform="scale(.26459)"/><path style="fill:#fff;stroke-width:0;stroke-linecap:round;stroke-linejoin:round" d="M182.543 201.654c-12.765-3.2-17.2-18.717-7.878-27.562 10.132-9.613 27.175-2.348 27.139 11.569-.012 4.75-1.385 8.057-4.736 11.407-3.915 3.915-9.794 5.772-14.525 4.586z" transform="scale(.26459)"/><path style="fill:#fff;stroke-width:0;stroke-linecap:round;stroke-linejoin:round" d="M182.543 201.64c-12.735-3.101-17.197-18.706-7.878-27.548 10.157-9.637 27.134-2.365 27.154 11.631.011 7.467-4.896 13.744-12.234 15.647-3.86 1.002-4.01 1.007-7.042.27z" transform="scale(.26459)"/><path style="fill:#fff;stroke-width:0;stroke-linecap:round;stroke-linejoin:round" d="M182.543 201.64c-12.735-3.101-17.197-18.706-7.878-27.548 10.157-9.637 27.134-2.365 27.154 11.631.011 7.316-4.565 13.443-11.608 15.545-3.695 1.102-4.504 1.141-7.668.371z" transform="scale(.26459)"/><path style="fill:#fff;stroke-width:0;stroke-linecap:round;stroke-linejoin:round" d="M175.89 237.961c-20.035-3.992-36.346-19.295-41.617-39.044-1.82-6.817-1.974-18.517-.333-25.254 4.676-19.196 20.51-35.046 39.672-39.71 6.79-1.654 18.465-1.506 25.305.32 20.112 5.367 35.144 21.615 39.11 42.27 2.344 12.203-.257 25.717-7.136 37.084-3.471 5.737-11.947 14.084-17.873 17.603-11.288 6.704-24.891 9.17-37.129 6.731zm19.716-15.296c12.946-3.338 23.606-14.077 27.135-27.337 1.198-4.505 1.319-13.023.256-18.12-1.852-8.882-7.843-17.664-15.638-22.925-22.319-15.063-52.692-3.247-58.842 22.891-4.672 19.854 8.053 40.739 27.791 45.615 4.949 1.223 14.309 1.162 19.298-.124z" transform="scale(.26459)"/></svg>
```

Comment:
A few minor issues with the Diveball:

![image](https://user-images.githubusercontent.com/7288322/149603069-abc18923-df2f-4fcc-964d-6538212d1796.png)
- The light blue seems to be overlapping the black outline, would probably be better to just apply a black line color to the blue area, instead of manually adding the outline.
- The light blue area doesn't quite extend into the corners.
- The dark blue area down the bottom doesn't align with the black line (apply the line color same as above for light blue area)
- Dark blue area down the bottom is a bit jagged, I think there's probably a few too many points in it (note that you can drag the "handles" to increase/decrease the curve (both sides of the handles should align though for a less jagged curve))
  - ![image](https://user-images.githubusercontent.com/7288322/149603260-de706ba2-a886-42b4-9ac4-ae36ebbc47e9.png)
- Not sure what's happened with the right line on the dark blue, seems to not be a single thickness:
  - ![image](https://user-images.githubusercontent.com/7288322/149603359-f71db7d4-7083-4183-9fee-fe4a3267ad0b.png)
- There seems to be a big black blob off to the side/under the main image 😅 (although out of frame, still shouldn't be there)
  - ![image](https://user-images.githubusercontent.com/7288322/149603278-4f000f47-9cbe-412f-9627-65fc40c4c87a.png)

If you need any tips with fixing those up, let me know on the Discord and I'll try my best to help you out 👍

Reply:
Sorry, it's called "stroke" for the outlines, so set the stroke color to black, and adjust from there.